src/traces/histogram/attributes.js

```diff
@@ -82,15 +82,21 @@ module.exports = {
         role: 'style',
         description: [
             'Determines whether or not the x axis bin attributes are picked',
-            'by an algorithm.'
+            'by an algorithm. Note that this should be set to false if you',
```

Comment:
@etpinard yep agreed -- I'll try to think of wording to express the logic ( https://github.com/plotly/documentation/issues/428 ). I really want to change that because I think that having 1 less variable to worry about to get a result makes these 'trickier' attributes w/ dependents more approachable for new users.